vulture_os/toolkit/api_parser/crowdstrike/crowdstrike.py

```diff
@@ -311,3 +327,40 @@ def test(self):
                 "status": False,
                 "error": str(e)
             }
+
+
+    def execute(self):
+        self.login() # establish a session to console
+
+        kinds = ["detections"]
+        if self.request_incidents:
+            kinds.append("incidents")
+
+        for kind in kinds:
+            # Default timestamp is 24 hours ago
+            since = self.last_collected_timestamps.get(f"crowdstrike_falcon_{kind}") or (timezone.now() - timedelta(days=7))
```

Comment:
switching from `last_api_call` to `last_collected_timestamps` use without handling any migration from old version to the next